modules/table/src/main/java/org/apache/ignite/internal/table/distributed/replicator/PartitionReplicaListener.java

```diff
@@ -765,14 +791,23 @@ private CompletableFuture<Object> processMultiEntryAction(ReadWriteMultiRowRepli
                         RowId lockedRowId = deleteLockFuts[futNum++].join();
 
                         if (lockedRowId != null) {
-                            rowIdsToDelete.add(lockedRowId);
+                            rowIdsToDelete.put(new UUID(lockedRowId.mostSignificantBits(), lockedRowId.leastSignificantBits()), null);
```

Comment:
I also believe that RowId has `uuid` method for your convenience

Reply:
Yes, I replaced one found occurrence with a shorter constructor.